src/main/java/org/osiam/OsiamHome.java

```diff
@@ -1,10 +1,8 @@
 package org.osiam;
 
 import com.google.common.base.Strings;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Value;
+import org.slf4j.*;
```

Comment:
My comment about wildcard imports from the file `Osiam.java` holds true here as well.